assets/chef-recipes/cookbooks/mdbe_build/recipes/default.rb

```diff
@@ -400,20 +400,22 @@
       action :install
     end
   end
-  execute 'disable MariaDB repo' do
-    command "sudo mv /etc/apt/sources.list.d/mariadb.list /etc/apt/sources.list.d/mariadb.list.save"
-  end
-  apt_update 'update apt cache' do
-    action :update
-  end
-  execute 'install dependencies mariadb-server' do
-    command "apt-get --fix-broken --yes build-dep --quiet --allow-downgrades mariadb-server --target-release #{node.attributes['lsb']['codename']}"
-  end
-  execute 'enable MariaDB repo' do
-    command "sudo mv /etc/apt/sources.list.d/mariadb.list.save /etc/apt/sources.list.d/mariadb.list"
-  end
-  apt_update 'update apt cache' do
-    action :update
+  if File.file?('/etc/apt/sources.list.d/mariadb.list')
+    execute 'disable MariaDB repo' do
+      command "sudo mv /etc/apt/sources.list.d/mariadb.list /etc/apt/sources.list.d/mariadb.list.save"
+    end
+    apt_update 'update apt cache' do
+      action :update
+    end
+    execute 'install dependencies mariadb-server' do
```

Comment:
Your change simply removes the installation of MariaDB Server dependencies. They must be installed. It's the core point of this recepie.